app/assets/javascripts/angular/angular-config.js.erb

```diff
@@ -3,7 +3,7 @@ var app = angular.module('fritter', ['ngRoute', 'ui.bootstrap', 'wu.masonry','in
 app.config(['$routeProvider',function($routeProvider){
   $routeProvider
   .when('/',          {templateUrl: "<%= asset_path('home.html') %>",     controller: 'HomeCtrl'})
-  .when('/user/',     {templateUrl: "<%= asset_path('user.html') %>",     controller: 'UserCtrl'})
+  .when('/user/:id',     {templateUrl: "<%= asset_path('home.html') %>",     controller: 'HomeCtrl'})
```

Comment:
You probably don't need to pass in the user id through the url, since you're using UserServices. the url doesn't look so nice now, unless you allow the current user to look at other users' pages/pins